api/constants/constants.go

```diff
@@ -19,6 +19,7 @@ package constants
 
 import (
 	"encoding/json"
+	"net/http"
```

Comment:
Can we avoid importing the entire HTTP package for a few constants?

With this change, any program that needs to import `constants` to pull in a simple value now imports an entire http server and client, which can cause in binary bloat.